LibertyDSNP/parquetjs
# Feature: Timestamp support for JSONSchema generated schemas

Problem
=======

Closes: #93 

Solution
========

Add checks for `format == 'date-time`` inside the string type check for`string` and `array of string`

Change summary:
---------------
* Added format check for string fields ( and string arrays )
* Check for JSONSchema property `format` with value `date-time`
* Updated jsonschema tests and updated the snapshots

Steps to Verify:
----------------
1. Generate a JSON Schema or use an existing one
2. Add `"format":"date-time"` to the field which should have a Date/Time value
3. Ensure that the value is a valid `Date` object
4. Enjoy
